integration-tests/cli/state-import.it.js

```diff
@@ -0,0 +1,131 @@
+import { oneLine } from 'common-tags'
+import StateImport from '@commercetools/state-importer'
+import { getCredentials } from '@commercetools/get-credentials'
+import { clearData } from './helpers/utils'
+import states from '../../packages/state-importer/test/helpers/sampleStates.json'
+
+let projectKey
+if (process.env.CI === 'true') projectKey = 'state-import-integration-test'
+else projectKey = process.env.npm_config_projectkey
+
+describe('State importer', () => {
+  let apiConfig
+  let stateImport
+
+  const logger = {
+    error: () => {},
+    warn: () => {},
+    info: () => {},
+    debug: () => {},
+  }
+
+  beforeAll(() =>
+    getCredentials(projectKey).then(credentials => {
+      apiConfig = {
+        host: 'https://auth.sphere.io',
+        apiUrl: 'https://api.sphere.io',
+        projectKey,
+        credentials,
+      }
+      // Clear all existing states
+      return clearData(apiConfig, 'states')
+    })
+  )
+
+  afterAll(() => clearData(apiConfig, 'states'))
+
+  beforeEach(() => {
+    stateImport = new StateImport({ apiConfig }, logger)
+  })
+
+  test('should create states on the CTP', async () => {
```

Comment:
Why `test` and not `it`? Is that a pattern for integration tests in the repo?

Reply:
it actually makes sense. I'll change it, and create another PR for the other int tests. There seems to be different uses everywhere